src/ariel-os-macros/src/config.rs

```diff
@@ -134,13 +139,15 @@ mod config_macro {
 
     #[derive(Debug, enum_iterator::Sequence)]
     pub enum ConfigKind {
+        Ble,
```

Comment:
Now that I think of think, we cannot have that variant defined yet, otherwise the error message advertises a feature that's not supported yet:
```suggestion
        // Ble,
```
Another option would be to have `enum_iterator` skip that variant, but I could not find that from a quick glance at its documentation.